__test__/pet.test.js

```diff
@@ -0,0 +1,155 @@
+const Pet = require('../src/pet');
+
+describe('constructor', () => {
+    it('returns an object', () => {
+        expect(new Pet('Fido')).toBeInstanceOf(Object);
+    });
+});
+
+describe('constructor', () => {
+    it('sets the name property', () => {
+        const pet = new Pet('Fido');
+        expect(pet.name).toEqual('Fido');
+    });
+});
+
+describe('constructor', () => {
+    it('has an initial age of 0', () => {
+        const pet = new Pet('Fido');
+        expect(pet.age).toEqual(0);
+    });
+});
+
+describe('constructor', () => {
+
+    it('throws an error if the pet is not alive', () => {
+        const pet = new Pet('Fido');
+
+        pet.age = 30;
+
+        expect(() => pet.growUp()).toThrow('Your pet is no longer alive :(');
+    });
+
+    it('increments the age by 1', () => {
+        const pet = new Pet('Fido');
+        pet.growUp();
+        expect(pet.age).toEqual(1);
+    });
+});
+
+describe('constructor', () => {
+    it('increases the hunger property by 5', () => {
+        const pet = new Pet('Fido');
+        pet.growUp();
+        expect(pet.hunger).toEqual(5);
+    });
+});
+
+describe('constructor', () => {
+    it('decreases fitness by 3', () => {
+        const pet = new Pet('Fido');
+        pet.fitness = 10;
+        pet.growUp();
+        expect(pet.fitness).toEqual(7);
+    });
+});
```

Comment:
a describe block should be used to combine similar tests together. You should basically never have only one test in a describe block. The string you pass as the first argument to a describe block should describe how these tests are related. These tests here should be moved combined together and given more appropriate descriptions, e.g.
```
describe('constructor', () => {
  it('returns an object', () => {
    expect(new Pet('Fido')).toBeInstanceOf(Object);
  });

  it('sets the name property', () => {
      const pet = new Pet('Fido');
      expect(pet.name).toEqual('Fido');
  });

  it('has an initial age of 0', () => {
      const pet = new Pet('Fido');
      expect(pet.age).toEqual(0);
  });
  
  //should also add tests in this describe block for the other instantiated properties, i.e. a test for hunger having an initial value of 0, fitness having an initial value of 10 and children having an initial value of an empty array
});

describe('growUp', () => {
  it('throws an error if the pet is not alive', () => {
    const pet = new Pet('Fido');
    pet.age = 30;

    expect(() => pet.growUp()).toThrow('Your pet is no longer alive :(');
  });

  it('increments the age by 1', () => 
    const pet = new Pet('Fido');
    pet.growUp();

    expect(pet.age).toEqual(1);
  });

  it('increases the hunger property by 5', () => {
    const pet = new Pet('Fido');
    pet.growUp();
    
    expect(pet.hunger).toEqual(5);
  });

  it('decreases fitness by 3', () => {
    const pet = new Pet('Fido');
    pet.fitness = 10;
    pet.growUp();

    expect(pet.fitness).toEqual(7);
  });
});
```